docs/source/intro.rst

```diff
@@ -16,4 +16,14 @@ This means that all data is uniquely addressable from the associated parameters.
 With **signac**, we define the processes generating and manipulating a specific data set as a sequence of operations on a job.
 Using this abstraction, **signac** can define workflows on an arbitrary **signac** data space.
 
-.. image:: images/signac_data_space.png
+.. _concepts_overview
+.. figure:: images/signac_data_space.png
+
+    Overview of the **signac** framework.
+    When a project is created, a workspace directory is intialized on disk.
+    Each job is uniquely identified by a set of user-defined state points, which are formatted as dictionaries.
+    The workspace holds a directory for each job, containing JSON files that store the state point and job document.
+    The job directory name is a hash of the state point's contents.
+    Here, the ``init.py`` file initializes an empty project and adds one job with state point ``{"a": 1}``.
+    Next, users define a workflow using a subclass of **signac-flow**'s ``FlowProject``.
+    The workflow shown has three operations (simulate, analyze, visualize) that, when executed, produce two new files ``results.txt`` and ``plot.png`` in the job directory.
```

Comment:
May rephrase this to be more general. We might want to talk about flow in the context of how they fit together, rather than as the next step.

Reply:
Great improvement for now